config/frontend_redirect.yml.hudson

```diff
@@ -0,0 +1,15 @@
+development:
+  base: "http://localhost:2727"
+  password_reset: "/#/reset-password"
+  
+test:
+  base: "http://localhost:2727"
+  password_reset: "/#/reset-password"
+  
+staging:
+  base: "http://demo.zooniverse.org/panoptes-front-end"
+  password_reset: "/#/reset-password"
+  
+production:
+  base: "https://panoptes.zooniverse.org/"
```

Comment:
I don't think those will necessarily be used. I was including them as examples. We'll need to add a config file on s3 whenever we decide on deployment locationa